### ISSUE TITLE
As a user I'd like to import and export to common recipe formats

### ISSUE THREAD

**Issue by JavaZauber** [JackMiranda/herms#116]
Managing recipes on your computer has [not been invented here](https://en.wikipedia.org/wiki/Not_invented_here).
As a user I'd like to import common recipe formats (maybe export as a separate ticket?).

The most common formats are

- CookMaster
- Rezkonv (RK Format)
- Mealmaster (MM Format)

For more technical information please refer to https://recipetools.sourceforge.net/joomla3/index.php/cb2cb

**Comment by langston-barrett** [comment]
Absolutely! Thanks for doing some research and gathering some links. This isn't a big priority for me personally, but I'm happy to help out and provide advice if anyone else wants to tackle this.

**Comment by JavaZauber** [comment]
Up until then I'm probably going to store everything inside .txt files
I'm most likely going to switch to https://grocy.info/ anyway
